charger/plugchoice/const.go

```diff
@@ -0,0 +1,13 @@
+package plugchoice
+
+// Connector status values
+const (
```

Comment:
Seems these can be removed, use ocpp/core.ChargePointStatus instead to avoid duplication.